sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/UrlHelper.cs

```diff
@@ -0,0 +1,70 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+
+namespace OpenTelemetry.Exporter.AzureMonitor
+{
+    internal class UrlHelper
+    {
+        private const string SchemePostfix = "://";
+        private const string Colon = ":";
+        private const string HttpPort80 = "80";
+        private const string HttpPort443 = "443";
+
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        internal static string GetUrl(Dictionary<string, string> tags)
+        {
+            if (tags.TryGetValue(SemanticConventions.AttributeHttpUrl, out var url))
+            {
+                Uri.TryCreate(url, UriKind.RelativeOrAbsolute, out var uri);
+
+                if (uri?.IsAbsoluteUri == true)
```

Comment:
would the null check be necessary here if you include the bool return from the TryCreate as part of if statement?

if (
```suggestion
                if (Uri.TryCreate(url, UriKind.RelativeOrAbsolute, out var uri) 
                    && uri.IsAbsoluteUri == true)
```